### ISSUE TITLE
multiple refunds on the same payment fail

### ISSUE THREAD

**Issue by haayman** [mollie/mollie-api-node#256]
I'm pretty sure it worked before that it was possible to have multiple refunds on a single payment until the amount is zero.

I've got a ticket application where it is possible to cancel tickets. Right now you can only cancel tickets once, but before it was possible to cancel all tickets one at the time, creating a new refund on the same paymen. So why can't we have multiple refunds anymore?

**Comment by sjorsvanheuveln** [comment]
Since when did you start having issues with this? I have another issue, but my payment system has been broken since 18th of jan and was working 4 years straight without issues.

**Comment by haayman** [comment]
I don't know. Since the tickets are for a theater group and due to the pandemic I haven't worked on the system for a long time. 
Looking through my tests I notice I don't have that test-case unfortunately, but I'm convinced this was possible.

**Comment by Pimm** [comment]
Thanks for opening the issue.

According to our changelog, [since 17 March 2020](https://docs.mollie.com/changelog/v2/changelog#thursday-12th) "_[d]uplicate Refunds (same amount as another Refund on the same Payment within the hour) are now blocked by the API._" Could this be affecting you? I can imagine that if you cancel two tickets individually, the refunds are for the same amount.

I was not personally involved in this change, nor do I know whether there is a workaround. I will ask around internally.

**Comment by Pimm** [comment]
Could you verify that the error you are receiving is:
> ApiError: A duplicate refund has been detected

**Comment by haayman-imagem** [comment]
yes, that makes a lot of sense. I will ask. Thanks a lot
